src/renderer/actions.ts

```diff
@@ -2395,6 +2378,99 @@ export const OPEN_MESSAGE_BOX = 'OPEN_MESSAGE_BOX';
 export const CLOSE_MESSAGE_BOX = 'CLOSE_MESSAGE_BOX';
 
 
+/**
+ * Bring up File upload/download dialogs
+ */
+
+export function fileUploadInteractive() {
+    return showDialog('fileUploadDialog');
+}
+
+
+export function fileDownloadInteractive() {
+    return showDialog('fileDownloadDialog');
+}
+
+
+/**
+ *
+ * @param dir Directory to be used
+ * @param file File to upload
+ */
+
```

Comment:
Remove action doc, or complete. If latter, remove empty line between docs and action.